src/NuGet.Clients/NuGet.PackageManagement.UI/Options/ConfigurationFilesViewModel.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Windows.Input;
+using Microsoft.VisualStudio.ComponentModelHost;
+using Microsoft.VisualStudio.PlatformUI;
+using Microsoft.VisualStudio.Services.Common;
+using NuGet.ProjectManagement;
+using NuGet.VisualStudio;
+
+namespace NuGet.PackageManagement.UI.Options
+{
+    public class ConfigurationFilesViewModel
+    {
+        public ObservableCollection<string> ConfigurationFilesCollection { get; private set; }
+        public string SelectedPath { get; set; }
+        public ICommand OpenConfigurationFile { get; set; }
```

Comment:
This property is initialized in the constructor and hence I think it can be marked as `readonly`.

Reply:
Since this is a property, the way to achieve this would be to remove the setter:

public ICommand OpenConfigurationFile { get; }